go.mod

```diff
@@ -2,14 +2,15 @@ module github.com/codeready-toolchain/toolchain-e2e
 
 require (
 	github.com/codeready-toolchain/api v0.0.0-20200608101253-f0555f65f63d
+	github.com/codeready-toolchain/host-operator v0.0.0-00010101000000-000000000000
```

Comment:
We managed to avoid adding dependencies to the operator source code in our e2e tests so far. It seems that you only need it for `MaxPoolSize`. Let's hardcode the expected value in the tests instead. It would be better to hard code it in the tests anyway. Just to make sure we have the expected configuration (hardcoded atm) in the operator.

Reply:
sure, let me do that

Reply:
done in 42bea0b